sp800-63a/sec4_ial.md

```diff
@@ -109,7 +109,7 @@ At a minimum, the applicant must be verified by a process that is able to achiev
 
 #### 4.4.1.5. Presence Requirements
 
-The CSP SHOULD perform identity proofing in-person. The CSP MAY perform remote identity proofing. The CSP SHOULD offer both in-person and remote proofing.
+In-person and remote identity proofing are allowed. The CSP SHOULD offer both in-person and remote proofing.
```

Comment:
Frankly, I think we are fooling ourselves that even making that statement means it will happen, so I am ok with the text as-is. However, if you have language that you would recommend, that doesn't confuse like our last attempt which demonstrated preference, please propose.